homework7/hw7task3/project3_/tic_tac_toe_checker.py

```diff
@@ -0,0 +1,47 @@
+"""
+Homework 7 task3.
+
+Given a Tic-Tac-Toe 3x3 board (can be unfinished).
+Write a function that checks if the are some winners.
+If there is "x" winner, function should return "x wins!"
+If there is "o" winner, function should return "o wins!"
+If there is a draw, function should return "draw!"
+If board is unfinished, function should return "unfinished!"
+Example:
+    [[-, -, o],
+     [-, x, o],
+     [x, o, x]]
+    Return value should be "unfinished"
+    [[-, -, o],
+     [-, o, o],
+     [x, x, x]]
+     Return value should be "x wins!"
+"""
+from typing import List
+
+
+def tic_tac_toe_checker(board: List[List]) -> str:
+    """Checks if the are some winners.
+
+    If there is "x" winner, function should return "x wins!"
+    If there is "o" winner, function should return "o wins!"
+    If there is a draw, function should return "draw!"
+    If board is unfinished, function should return "unfinished!".
+    """
+
+    all_possible_variants = board
+    diag_from_left = [item[i] for i, item in enumerate(board)]
+    diag_from_right = [item[-i - 1] for i, item in enumerate(board)]
+    rows = [list(item) for item in list(zip(*board))]
```

Comment:
Can we find a better name for rows?
Why do you need the second list?

Reply:
Fixed with rows. 

Yes, we really don`t need second list. Removed.
Firstly I used this list() because zip(*board) return zip object.